src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -324,12 +356,14 @@ private void scrollAndHighlightText(Text targetText) {
      */
     private void highlightText(Text targetText) {
         if (lastHighlighted != null) {
-            VBox prevBox = (VBox) lastHighlighted.getParent();
-            prevBox.setBackground(null);
+            HBox prevBox = (HBox) lastHighlighted.getParent();
+            prevBox.setBackground(new Background(new BackgroundFill(
+                    Color.rgb(255, 152, 36, 0.4), new CornerRadii(15), new Insets(16, 16, 0, 16))));
         }
         // type-casted to Vbox as every Text in helpWindow is wrapped by a Vbox as its parent
-        VBox vBox = (VBox) targetText.getParent();
-        Background highlight = new Background(new BackgroundFill(Color.LIGHTBLUE, CornerRadii.EMPTY, null));
+        HBox vBox = (HBox) targetText.getParent();
+        Background highlight = new Background(new BackgroundFill(
+                Color.rgb(255, 40, 40, 0.2), new CornerRadii(15), new Insets(16, 16, 0, 16)));
```

Comment:
Can consider using HelpWindow.css to edit the style of HelpWindow to reduce lines of code in HelpWindow.java